logs-shipper/src/main/java/com/ServerModule.java

```diff
@@ -17,6 +20,13 @@ public LogsConfiguration getLogsConfiguration() {
         return Configuration.getLogsConfiguration();
     }
 
+    @Provides
+    public RestHighLevelClient getRestHighLevelClient() {
+        return new RestHighLevelClient(
+                RestClient.builder(
+                        new HttpHost("elasticsearch", 9200, "http")));
```

Comment:
Nice to have the elasticsearch host and port inside your configuration file

Reply:
done